apps/vmq_server/src/vmq_queue.erl

```diff
@@ -221,6 +221,8 @@ online({add_session, SessionPid, #{allow_multiple_sessions := false} = Opts}, Fr
     %% we've to disconnect currently attached sessions
     %% and wait with the reply until all the sessions
     %% have been disconnected
+    #state{id = SubscriberId} = State,
+    lager:warning("SubscriberId ~p disconnected by same client_id", [SubscriberId]),
```

Comment:
This should be either a `info` or a `debug` as being disconnected is not an exceptional event but a part of a normal flow. I personally would say it should be on the `debug` level.

Reply:
To conform with other messages it should start with a lower case letter and be on the form we generally use "<action> due to <reason>", so something like: `lager:debug("client ~p disconnected due to multiple sessions not allowed", [SubscriberId])` or something like that, though I'm not 100% sure if the reason I just gave is the most descriptive.